### ISSUE TITLE
Network cards not showing internal IP

### ISSUE THREAD

**Issue by GibStorm** [ArcadeRenegade/SidebarDiagnostics#243]
**1. Your Operating System**
Windows 10 Pro, 1903

**2. App Version**
3.5.3

**3. Any Relevant Settings**
All options for network card monitoring are active. Thus, both internal and external IP should be shown. Have tried disabling external IP, disabling internal IP and enabling it again etc, to no avail.

**4. Steps to Reproduce**
Activate showing internal IP, see that it doesn't work.

![image](https://user-images.githubusercontent.com/4786264/63743850-d88c3a00-c89d-11e9-8da3-c3e5dcb97bc1.png)
![Annotation 2019-08-27 074041](https://user-images.githubusercontent.com/4786264/63743909-03768e00-c89e-11e9-8851-467a9fd1bbd9.png)

**Comment by GeekJosh** [comment]
Do you have any virtual adapters? I do (created by Hyper-V) and the adapter they're actually using for connectivity does not show it's internal IP

**Comment by unRODbar** [comment]
I just discovered the same issue on my machine, but it only happened when I changed the name of the adapter under Hardware in the Sidebar settings. 
For example my full Wi-Fi adapter device name is "Intel(R) Wi-Fi 6 AX200 160MHz" and changing only one character of this string in the settings and the internal IP disappears, reverting the name to its default and the IP appears again.
Also issue #146 from 2017 appears to be the same problem.
